azurerm/internal/services/eventgrid/parse/eventgrid_event_subscription.go

```diff
@@ -25,14 +25,7 @@ func EventGridEventSubscriptionID(input string) (*EventGridEventSubscriptionId,
 
 	eventSubscription := EventGridEventSubscriptionId{
 		Scope: segments[0],
-	}
-
-	if eventSubscription.Name, err = id.PopSegment("eventSubscriptions"); err != nil {
-		return nil, err
-	}
-
-	if err := id.ValidateNoEmptySegments(input); err != nil {
-		return nil, err
+		Name:  segments[1],
```

Comment:
(for passers by) this appears to be a rebase containing a change made the other day - but this is expected 👍

Reply:
correct. it was just for testing purposes, as the other fix wasn't merged during the time i worked on this branch.